engine/src/main/java/io/camunda/zeebe/streamprocessor/StreamProcessorBuilder.java

```diff
@@ -128,8 +128,6 @@ private void validate() {
     Objects.requireNonNull(typedRecordProcessorFactory, "No stream processor factory provided.");
     Objects.requireNonNull(actorSchedulingService, "No task scheduler provided.");
     Objects.requireNonNull(streamProcessorContext.getLogStream(), "No log stream provided.");
-    Objects.requireNonNull(
```

Comment:
This line will be removed due to lifecycle changes (i.e. the field will now be set at a later time).